Flash Chat iOS13/Controllers/ThreadListViewController.swift

```diff
@@ -76,43 +76,55 @@ class ThreadListViewController: UITableViewController {
 		var textField = UITextField()
 		let cancelAction = UIAlertAction(title: "Cancel", style: .cancel)
 		let addAction = UIAlertAction(title: "Add", style: .default) { [self] (action) in
+			guard let text = textField.text else { return }
+			if text.contains(" ") {
+				let spacesNotAllowed = UIAlertController(title: "The recipients field can't contain spaces. Please try again with the format \"name@example.com,name@example.com\".", message: nil, preferredStyle: .alert)
+				let okAction = UIAlertAction(title: "OK", style: .default)
+				spacesNotAllowed.addAction(okAction)
```

Comment:
💬 
From a UX perspective, it'd be nicer to discard spaces rather than warn. 
But I'm not concerned with this at this time.

Reply:
Will update to use replacingOccurrences(of:with:).